### PR TITLE
updating links from fwlink to real link if exists

### PR DESCRIPTION
**Pull request by eddynaka**
updating links from fwlink to real link if exists - contributes to #3391

### CODE REVIEW

File: docs/framework/wcf/feature-details/peer-to-peer-networking.md [hunk 1]
```diff
@@ -21,4 +21,4 @@ Peer Channel is a multiparty, peer-to-peer (P2P) communication technology in Win
  [Peer Channel Custom Peer Resolver](https://docs.microsoft.com/previous-versions/dotnet/netframework-3.5/ms751466(v=vs.90))  
   
 ## Peer Channel Team blog  
- [Peer Channel Team Blog](https://go.microsoft.com/fwlink/?LinkID=114530)
+ [Peer Channel Team Blog](https://docs.microsoft.com/archive/blogs/)
```
Comment (Youssef1313) [inline docs/framework/wcf/feature-details/peer-to-peer-networking.md]:
```suggestion
 [Peer Channel Team Blog](https://docs.microsoft.com/archive/blogs/peerchan/)
```

File: docs/framework/wcf/feature-details/secure-conversations-and-secure-sessions.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ ms.assetid: 48cb104a-532d-40ae-aa57-769dae103fda
...
+ The bootstrap policy (defined in section 6.2.7 of the [WS-SecurityPolicy](http://docs.oasis-open.org/ws-sx/ws-securitypolicy/200702/ws-securitypolicy-1.2-spec-os.html) standard) contains the message security assertions used to secure the channel and authenticate the client prior to the RST/SCT and RSTR/SCT exchange. Certain WCF standard bindings have a `Security.Message.EstablishSecurityContext` property which controls whether secure conversation is used. When using custom bindings the bootstrap is indicated by nesting security binding elements, either through [\<secureConversationBootstrap>](../../../../docs/framework/configure-apps/file-schema/wcf/secureconversationbootstrap.md) in the configuration file, or by calling <xref:System.ServiceModel.Channels.SecurityBindingElement.CreateSecureConversationBindingElement%2A> in code.
```
Comment (Youssef1313) [inline docs/framework/wcf/feature-details/secure-conversations-and-secure-sessions.md]: The site supports https.

@mairaw, I don't know if you'd like to update the internal fwlink to redirect to https.

```suggestion
 The bootstrap policy (defined in section 6.2.7 of the [WS-SecurityPolicy](https://docs.oasis-open.org/ws-sx/ws-securitypolicy/200702/ws-securitypolicy-1.2-spec-os.html) standard) contains the message security assertions used to secure the channel and authenticate the client prior to the RST/SCT and RSTR/SCT exchange. Certain WCF standard bindings have a `Security.Message.EstablishSecurityContext` property which controls whether secure conversation is used. When using custom bindings the bootstrap is indicated by nesting security binding elements, either through [\<secureConversationBootstrap>](../../../../docs/framework/configure-apps/file-schema/wcf/secureconversationbootstrap.md) in the configuration file, or by calling <xref:System.ServiceModel.Channels.SecurityBindingElement.CreateSecureConversationBindingElement%2A> in code.  
```

File: docs/framework/wcf/feature-details/system-web-routing-integration.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ ms.date: "03/30/2017"
...
+When hosting a Windows Communication Foundation (WCF) service in Internet Information Service (IIS) you place a .svc file in the virtual directory. This .svc file specifies the service host factory to use as well as the class that implements the service. When making requests to the service you specify the .svc file in the URI, for example: `http://contoso.com/EmployeeServce.svc`. For programmers writing REST services this type of URI is not optimal. URIs for REST services specify a specific resource and normally do not have any extensions. The <xref:System.Web.Routing> integration feature allows you to host a WCF REST service that responds to URIs without an extension. For more information about routing see [ASP.NET Routing](https://docs.microsoft.com/previous-versions/cc668201(v=vs.140)).
```
Comment (Youssef1313) [inline docs/framework/wcf/feature-details/system-web-routing-integration.md]: Should this be changed to point to the current version from asp documentation ?

File: docs/framework/wcf/feature-details/system-web-routing-integration.md [hunk 4]
```diff
@@ -66,4 +66,4 @@ When hosting a Windows Communication Foundation (WCF) service in Internet Inform
 ## See also
 
 - [WCF Web HTTP Programming Model](../../../../docs/framework/wcf/feature-details/wcf-web-http-programming-model.md)
-- [ASP.NET Routing](https://go.microsoft.com/fwlink/?LinkId=184660)
+- [ASP.NET Routing](https://docs.microsoft.com/previous-versions/cc668201(v=vs.140))
```
Comment (Youssef1313) [inline docs/framework/wcf/feature-details/system-web-routing-integration.md]: Same comment

File: docs/framework/wcf/feature-details/transaction-protocols.md [hunk 5]
```diff
@@ -8,7 +8,7 @@ Windows Communication Foundation (WCF) implements WS-Atomic Transaction and WS-C
   
 |Specification/Document|Version|Link|  
 |-----------------------------|-------------|----------|  
-|WS-Coordination|1.0<br /><br /> 1.1|<https://go.microsoft.com/fwlink/?LinkId=96104><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96079](https://go.microsoft.com/fwlink/?LinkId=96079)|  
+|WS-Coordination|1.0<br /><br /> 1.1|<https://go.microsoft.com/fwlink/?LinkId=96104><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96079](http://docs.oasis-open.org/ws-tx/wscoor/2006/06)|
```
Comment (Youssef1313) [inline docs/framework/wcf/feature-details/transaction-protocols.md]: Link text needs to be updated + use https.

File: docs/framework/wcf/feature-details/transaction-protocols.md [hunk 6]
```diff
@@ -55,9 +55,9 @@ Windows Communication Foundation (WCF) implements WS-Atomic Transaction and WS-C
...
-|wscoor|1.0<br /><br /> 1.1|<https://go.microsoft.com/fwlink/?LinkId=96078><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96079](https://go.microsoft.com/fwlink/?LinkId=96079)|  
-|wsat|1.0<br /><br /> 1.1|<https://go.microsoft.com/fwlink/?LinkId=96080><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96081](https://go.microsoft.com/fwlink/?LinkId=96081)|  
-|t|Pre-1.3<br /><br /> 1.3|<https://go.microsoft.com/fwlink/?LinkId=96082><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96100](https://go.microsoft.com/fwlink/?LinkId=96100)|  
+|wscoor|1.0<br /><br /> 1.1|<https://go.microsoft.com/fwlink/?LinkId=96078><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96079](http://docs.oasis-open.org/ws-tx/wscoor/2006/06)|  
+|wsat|1.0<br /><br /> 1.1|<https://go.microsoft.com/fwlink/?LinkId=96080><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96081](http://docs.oasis-open.org/ws-tx/wsat/2006/06)|  
+|t|Pre-1.3<br /><br /> 1.3|<https://go.microsoft.com/fwlink/?LinkId=96082><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96100](http://docs.oasis-open.org/ws-sx/ws-trust/200512)|
```
Comment (Youssef1313) [inline docs/framework/wcf/feature-details/transaction-protocols.md]: Same comment

**Review by Youssef1313 — CHANGES_REQUESTED** [review]

File: docs/framework/wcf/feature-details/security-concepts-used-in-wcf.md [hunk 7]
```diff
@@ -9,18 +9,18 @@ Windows Communication Foundation (WCF) security is built upon concepts already i
...
   
 ## Industry-Wide Security Specifications  
   
 ### Public Key Infrastructure  
- Public Key Infrastructure (PKI) is a system of digital certificates, certification authorities, and other registration authorities that verify and authenticate each party involved in an electronic transaction through the use of public key cryptography. For more information, see [Windows Server 2008 R2 Certificate Services](https://go.microsoft.com/fwlink/?LinkId=210211).  
+ Public Key Infrastructure (PKI) is a system of digital certificates, certification authorities, and other registration authorities that verify and authenticate each party involved in an electronic transaction through the use of public key cryptography. For more information, see [Windows Server 2008 R2 Certificate Services](https://azure.microsoft.com/?ocid=cloudplat_hp).
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/security-concepts-used-in-wcf.md]:
```suggestion

Public Key Infrastructure (PKI) is a system of digital certificates, certification authorities, and other registration authorities that verify and authenticate each party involved in an electronic transaction through the use of public-key cryptography.
```

File: docs/framework/wcf/feature-details/security-concepts-used-in-wcf.md [hunk 8]
```diff
@@ -9,18 +9,18 @@ Windows Communication Foundation (WCF) security is built upon concepts already i
...
   
 ## Introduction to Security for WCF Web Services  
- The Microsoft Patterns and Practices group wrote an in-depth white paper on WCF security guidance which is available for download here: [WCF Security Guide](https://go.microsoft.com/fwlink/?LinkId=210210). This white paper describes the fundamental security concepts as they relate to web services, key WCF security concepts, intranet application scenarios, and internet application scenarios.  
+ The Microsoft Patterns and Practices group wrote an in-depth white paper on WCF security guidance which is available for download here: [WCF Security Guide](https://archive.codeplex.com/?p=wcfsecurityguide). This white paper describes the fundamental security concepts as they relate to web services, key WCF security concepts, intranet application scenarios, and internet application scenarios.
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/security-concepts-used-in-wcf.md]:
```suggestion

The Microsoft Patterns and Practices group wrote an in-depth white paper called [WCF Security Guide](https://archive.codeplex.com/?p=wcfsecurityguide). This white paper describes the fundamental security concepts as they relate to web services, key WCF security concepts, intranet application scenarios, and internet application scenarios.  
```

File: docs/framework/wcf/feature-details/security-guidance-and-best-practices.md [hunk 9]
```diff
@@ -6,7 +6,7 @@ helpviewer_keywords:
 ms.assetid: 79e4e6f7-0c00-4045-b0f0-e6a7c37c7788
 ---
 # Security Guidance and Best Practices
-The topics in this section present guidance for and items to consider when creating secure Windows Communication Foundation (WCF) applications. For more information about Windows Server AppFabric and security see, [Security Model for Windows Server App Fabric](https://go.microsoft.com/fwlink/?LinkID=201279&clcid=0x409)  
+The topics in this section present guidance for and items to consider when creating secure Windows Communication Foundation (WCF) applications. For more information about Windows Server AppFabric and security see, [Security Model for Windows Server App Fabric](https://docs.microsoft.com/previous-versions/appfabric/ee677202(v=azure.10))
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/security-guidance-and-best-practices.md]:
```suggestion

The articles in this section present guidance for and items to consider when creating secure Windows Communication Foundation (WCF) applications. For more information about Windows Server AppFabric and security, see [Security Model for Windows Server App Fabric](https://docs.microsoft.com/previous-versions/appfabric/ee677202(v=azure.10))  
```

File: docs/framework/wcf/feature-details/system-web-routing-integration.md (hunk 3 above)
Comment (mairaw) [reply docs/framework/wcf/feature-details/system-web-routing-integration.md]: which ones were you thinking @Youssef1313? My answer would be probably not. These are old articles so probably the .NET Framework version of those would be best. But I'd however change this to the version that has a TOC:
```suggestion
When hosting a Windows Communication Foundation (WCF) service in Internet Information Service (IIS) you place a .svc file in the virtual directory. This .svc file specifies the service host factory to use as well as the class that implements the service. When making requests to the service you specify the .svc file in the URI, for example: `http://contoso.com/EmployeeServce.svc`. For programmers writing REST services, this type of URI is not optimal. URIs for REST services specify a specific resource and normally do not have any extensions. The <xref:System.Web.Routing> integration feature allows you to host a WCF REST service that responds to URIs without an extension. For more information about routing see [ASP.NET Routing](https://docs.microsoft.com/previous-versions/aspnet/cc668201(v=vs.100)).  
```

File: docs/framework/wcf/feature-details/system-web-routing-integration.md (hunk 4 above)
Comment (mairaw) [reply docs/framework/wcf/feature-details/system-web-routing-integration.md]:
```suggestion
- [ASP.NET Routing](https://docs.microsoft.com/previous-versions/aspnet/cc668201(v=vs.100))
```

File: docs/framework/wcf/feature-details/transaction-protocols.md (hunk 5 above)
Comment (mairaw) [inline docs/framework/wcf/feature-details/transaction-protocols.md]: @eddynaka your tool is probably missing a few URLs that use the \<> syntax like in this sentence

```suggestion
|WS-Coordination|1.0<br /><br /> 1.1|<http://schemas.xmlsoap.org/ws/2004/10/wscoor/><br /><br /> <https://docs.oasis-open.org/ws-tx/wscoor/2006/06>|  
```

File: docs/framework/wcf/feature-details/transaction-protocols.md [hunk 10]
```diff
@@ -55,9 +55,9 @@ Windows Communication Foundation (WCF) implements WS-Atomic Transaction and WS-C
 |------------|-------------|-------------------|  
 |s11||<https://go.microsoft.com/fwlink/?LinkId=96014>|  
 |wsa|Pre-1.0<br /><br /> 1.0|http://www.w3.org/2004/08/addressing<br /><br /> <https://go.microsoft.com/fwlink/?LinkId=96022>|
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/transaction-protocols.md]:
```suggestion
|wsa|Pre-1.0<br /><br /> 1.0|`http://www.w3.org/2004/08/addressing`<br /><br /> <https://www.w3.org/2005/08/addressing/>|  
```

File: docs/framework/wcf/feature-details/transaction-protocols.md [hunk 11]
```diff
@@ -55,9 +55,9 @@ Windows Communication Foundation (WCF) implements WS-Atomic Transaction and WS-C
 |------------|-------------|-------------------|  
 |s11||<https://go.microsoft.com/fwlink/?LinkId=96014>|
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/transaction-protocols.md]:
```suggestion
|s11||<https://schemas.xmlsoap.org/soap/envelope/>|  
```

File: docs/framework/wcf/feature-details/transaction-protocols.md [hunk 12]
```diff
@@ -55,9 +55,9 @@ Windows Communication Foundation (WCF) implements WS-Atomic Transaction and WS-C
 |------------|-------------|-------------------|  
 |s11||<https://go.microsoft.com/fwlink/?LinkId=96014>|  
 |wsa|Pre-1.0<br /><br /> 1.0|http://www.w3.org/2004/08/addressing<br /><br /> <https://go.microsoft.com/fwlink/?LinkId=96022>|  
-|wscoor|1.0<br /><br /> 1.1|<https://go.microsoft.com/fwlink/?LinkId=96078><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96079](https://go.microsoft.com/fwlink/?LinkId=96079)|  
-|wsat|1.0<br /><br /> 1.1|<https://go.microsoft.com/fwlink/?LinkId=96080><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96081](https://go.microsoft.com/fwlink/?LinkId=96081)|  
-|t|Pre-1.3<br /><br /> 1.3|<https://go.microsoft.com/fwlink/?LinkId=96082><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96100](https://go.microsoft.com/fwlink/?LinkId=96100)|  
+|wscoor|1.0<br /><br /> 1.1|<https://go.microsoft.com/fwlink/?LinkId=96078><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96079](http://docs.oasis-open.org/ws-tx/wscoor/2006/06)|
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/transaction-protocols.md]:
```suggestion
|wscoor|1.0<br /><br /> 1.1|<http://schemas.xmlsoap.org/ws/2004/10/wscoor/><br /><br /> <https://docs.oasis-open.org/ws-tx/wscoor/2006/06>|  
```

File: docs/framework/wcf/feature-details/transaction-protocols.md [hunk 13]
```diff
@@ -55,9 +55,9 @@ Windows Communication Foundation (WCF) implements WS-Atomic Transaction and WS-C
 |------------|-------------|-------------------|  
 |s11||<https://go.microsoft.com/fwlink/?LinkId=96014>|  
 |wsa|Pre-1.0<br /><br /> 1.0|http://www.w3.org/2004/08/addressing<br /><br /> <https://go.microsoft.com/fwlink/?LinkId=96022>|  
-|wscoor|1.0<br /><br /> 1.1|<https://go.microsoft.com/fwlink/?LinkId=96078><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96079](https://go.microsoft.com/fwlink/?LinkId=96079)|  
-|wsat|1.0<br /><br /> 1.1|<https://go.microsoft.com/fwlink/?LinkId=96080><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96081](https://go.microsoft.com/fwlink/?LinkId=96081)|  
-|t|Pre-1.3<br /><br /> 1.3|<https://go.microsoft.com/fwlink/?LinkId=96082><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96100](https://go.microsoft.com/fwlink/?LinkId=96100)|  
+|wscoor|1.0<br /><br /> 1.1|<https://go.microsoft.com/fwlink/?LinkId=96078><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96079](http://docs.oasis-open.org/ws-tx/wscoor/2006/06)|  
+|wsat|1.0<br /><br /> 1.1|<https://go.microsoft.com/fwlink/?LinkId=96080><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96081](http://docs.oasis-open.org/ws-tx/wsat/2006/06)|
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/transaction-protocols.md]:
```suggestion
|wsat|1.0<br /><br /> 1.1|<http://schemas.xmlsoap.org/ws/2004/10/wsat/><br /><br /> <https://docs.oasis-open.org/ws-tx/wsat/2006/06>|  
```

File: docs/framework/wcf/feature-details/transaction-protocols.md (hunk 6 above)
Comment (mairaw) [reply docs/framework/wcf/feature-details/transaction-protocols.md]:
```suggestion
|t|Pre-1.3<br /><br /> 1.3|<http://schemas.xmlsoap.org/ws/2005/02/trust/><br /><br /> <https://docs.oasis-open.org/ws-sx/ws-trust/200512>|  
```

File: docs/framework/wcf/feature-details/transaction-protocols.md [hunk 14]
```diff
@@ -55,9 +55,9 @@ Windows Communication Foundation (WCF) implements WS-Atomic Transaction and WS-C
...
-|wscoor|1.0<br /><br /> 1.1|<https://go.microsoft.com/fwlink/?LinkId=96078><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96079](https://go.microsoft.com/fwlink/?LinkId=96079)|  
-|wsat|1.0<br /><br /> 1.1|<https://go.microsoft.com/fwlink/?LinkId=96080><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96081](https://go.microsoft.com/fwlink/?LinkId=96081)|  
-|t|Pre-1.3<br /><br /> 1.3|<https://go.microsoft.com/fwlink/?LinkId=96082><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96100](https://go.microsoft.com/fwlink/?LinkId=96100)|  
+|wscoor|1.0<br /><br /> 1.1|<https://go.microsoft.com/fwlink/?LinkId=96078><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96079](http://docs.oasis-open.org/ws-tx/wscoor/2006/06)|  
+|wsat|1.0<br /><br /> 1.1|<https://go.microsoft.com/fwlink/?LinkId=96080><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96081](http://docs.oasis-open.org/ws-tx/wsat/2006/06)|  
+|t|Pre-1.3<br /><br /> 1.3|<https://go.microsoft.com/fwlink/?LinkId=96082><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96100](http://docs.oasis-open.org/ws-sx/ws-trust/200512)|  
 |o||<https://go.microsoft.com/fwlink/?LinkId=96101>|
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/transaction-protocols.md]:
```suggestion
|o||<https://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-secext-1.0.xsd>|  
```

File: docs/framework/wcf/feature-details/transaction-protocols.md [hunk 15]
```diff
@@ -55,9 +55,9 @@ Windows Communication Foundation (WCF) implements WS-Atomic Transaction and WS-C
...
-|wsat|1.0<br /><br /> 1.1|<https://go.microsoft.com/fwlink/?LinkId=96080><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96081](https://go.microsoft.com/fwlink/?LinkId=96081)|  
-|t|Pre-1.3<br /><br /> 1.3|<https://go.microsoft.com/fwlink/?LinkId=96082><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96100](https://go.microsoft.com/fwlink/?LinkId=96100)|  
+|wscoor|1.0<br /><br /> 1.1|<https://go.microsoft.com/fwlink/?LinkId=96078><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96079](http://docs.oasis-open.org/ws-tx/wscoor/2006/06)|  
+|wsat|1.0<br /><br /> 1.1|<https://go.microsoft.com/fwlink/?LinkId=96080><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96081](http://docs.oasis-open.org/ws-tx/wsat/2006/06)|  
+|t|Pre-1.3<br /><br /> 1.3|<https://go.microsoft.com/fwlink/?LinkId=96082><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96100](http://docs.oasis-open.org/ws-sx/ws-trust/200512)|  
 |o||<https://go.microsoft.com/fwlink/?LinkId=96101>|  
 |xsd||<https://go.microsoft.com/fwlink/?LinkId=96102>|
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/transaction-protocols.md]:
```suggestion
|xsd||<https://www.w3.org/2001/XMLSchema>|  
```

File: docs/framework/wcf/feature-details/transport-security-with-certificate-authentication.md [hunk 16]
```diff
@@ -6,7 +6,7 @@ dev_langs:
...
...ation about X.509 certificates see [X.509 Public Key Certificates](/windows/desktop/SecCertEnroll/about-x-509-public-key-certificates). Certificates must be issued by a certification authority, which is often a third-party issuer of certificates. On a Windows Server domain, Active Directory Certificate Services can be used to issue certificates to client computers on the domain. For more information see [Windows 2008 R2 Certificate Services](https://azure.microsoft.com/?ocid=cloudplat_hp). In this scenario, the service is hosted under Internet Information Services (IIS) which is configured with Secure Sockets Layer (SSL). The service is configured with an SSL (X.509) certificate to allow clients to verify the identity of the server. The client is also configured with an X.509 certificate that allows the service to verify the identity of the client. The server’s certificate must be trusted by the client and the client’s certificate must be trusted by the server. The actual mechanics of how the service and client verifies each other’s identity is beyond the scope of this topic. For more information see [Digital Signature on Wikipedia](https://en.wikipedia.org/wiki/Digital_signature).
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/transport-security-with-certificate-authentication.md]:
```suggestion

This article discusses using X.509 certificates for server and client authentication when using transport security. For more information about X.509 certificates see [X.509 Public Key Certificates](/windows/desktop/SecCertEnroll/about-x-509-public-key-certificates). Certificates must be issued by a certification authority, which is often a third-party issuer of certificates. On a Windows Server domain, Active Directory Certificate Services can be used to issue certificates to client computers on the domain. In this scenario, the service is hosted under Internet Information Services (IIS) which is configured with Secure Sockets Layer (SSL). The service is configured with an SSL (X.509) certificate to allow clients to verify the identity of the server. The client is also configured with an X.509 certificate that allows the service to verify the identity of the client. The server’s certificate must be trusted by the client and the client’s certificate must be trusted by the server. The actual mechanics of how the service and client verifies each other’s identity is beyond the scope of this article. For more information, see [Digital Signature](https://en.wikipedia.org/wiki/Digital_signature) on Wikipedia.
```

File: docs/framework/wcf/feature-details/troubleshooting-queued-messaging.md [hunk 17]
```diff
@@ -109,7 +109,7 @@ For non-security related Web host issues refer to: [Web Hosting a Queued Applica
 
 **Q:** Where can I find answers to common questions on MSMQ?
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/troubleshooting-queued-messaging.md]:
```suggestion
```

File: docs/framework/wcf/feature-details/troubleshooting-queued-messaging.md [hunk 18]
```diff
@@ -109,7 +109,7 @@ For non-security related Web host issues refer to: [Web Hosting a Queued Applica
...
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/troubleshooting-queued-messaging.md]:
```suggestion
```

File: docs/framework/wcf/feature-details/troubleshooting-queued-messaging.md [hunk 19]
```diff
@@ -109,7 +109,7 @@ For non-security related Web host issues refer to: [Web Hosting a Queued Applica
...
 
-**A:** For more information about MSMQ, see [Microsoft Message Queuing](https://go.microsoft.com/fwlink/?LinkId=87810).
+**A:** For more information about MSMQ, see [Microsoft Message Queuing](https://www.microsoft.com/windowsserver2003/technologies/msmq/default.mspx).
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/troubleshooting-queued-messaging.md]:
```suggestion
```

File: docs/framework/wcf/feature-details/troubleshooting-queued-messaging.md [hunk 20]
```diff
@@ -109,7 +109,7 @@ For non-security related Web host issues refer to: [Web Hosting a Queued Applica
...
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/troubleshooting-queued-messaging.md]:
```suggestion
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @eddynaka! Left a few suggestions before we can merge this. 

Thanks @Youssef1313 for the help reviewing this as well.

File: docs/framework/wcf/feature-details/transaction-protocols.md [hunk 21]
```diff
@@ -8,7 +8,7 @@ Windows Communication Foundation (WCF) implements WS-Atomic Transaction and WS-C
   
 |Specification/Document|Version|Link|  
 |-----------------------------|-------------|----------|  
-|WS-Coordination|1.0<br /><br /> 1.1|<https://go.microsoft.com/fwlink/?LinkId=96104><br /><br /> [https://go.microsoft.com/fwlink/?LinkId=96079](https://go.microsoft.com/fwlink/?LinkId=96079)|  
+|WS-Coordination|1.0<br /><br /> 1.1|<http://schemas.xmlsoap.org/ws/2004/10/wscoor/><br /><br /> <https://docs.oasis-open.org/ws-tx/wscoor/2006/06>|  
 |WS-AtomicTransaction|1.0<br /><br /> 1.1|<https://go.microsoft.com/fwlink/?LinkId=96080><br /><br /> https://go.microsoft.com/fwlink/?LinkId=96081|
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/transaction-protocols.md]: Oh I missed these too

```suggestion
|WS-AtomicTransaction|1.0<br /><br /> 1.1|<http://schemas.xmlsoap.org/ws/2004/10/wsat/><br /><br /> <https://docs.oasis-open.org/ws-tx/wsat/2006/06>|  
```

**Review by mairaw — COMMENTED** [review]
One more comment. I missed one FwLink during the review.

**Review by mairaw — APPROVED** [review]
Thank you @eddynaka!

**Pull request merged**